client/scripts/views/components/widgets/user.ts

```diff
@@ -59,9 +59,11 @@ const User: m.Component<{
     const friendlyChainName = app.config.chains
       .getById(typeof user.chain === 'string' ? user.chain : user.chain?.id)?.name;
 
-    const adminsAndMods = app.chain
-      ? app.chain.meta.chain.adminsAndMods
-      : app.community ? app.community.meta.adminsAndMods : [];
+    // TODO: sometimes chain is not set yet.
```

Comment:
What do you suggest we do here?

Reply:
Just tested this. Happens when the user is not on any particular community page (e.g. the home page) and subsequently goes to their user profile. CW uses Edgeware as the default chain and thus edgeware takes a few 100 ms's to load, so we should wait until that happens. This check does that.